### ISSUE TITLE
🚨 [DEV-1165] Firebird critical issues: advanced search form and feedback modal

### ISSUE THREAD

**Issue by angelinanz** [hathitrust/firebird-common#71]
[DEV-1165]

[DEV-1165]: https://hathitrust.atlassian.net/browse/DEV-1165?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

- [x] 🚨 [1727510](https://axeauditor.dequecloud.com/test-run/b2613a94-fe0b-11ee-96ce-dfcb37e53e3c/issue/136cd630-fe3f-11ee-b5ef-37a650e942e8?sortField=ordinal&sortDir=asc&filter%5Bpage_number%5D=6&row=0) radio button does not have role and/or state
- [x] 🚨 [1727483](https://axeauditor.dequecloud.com/test-run/b2613a94-fe0b-11ee-96ce-dfcb37e53e3c/issue/848ccb44-fe3b-11ee-8829-bbb441666957?activeTab=dt-issue&page=0&pageSize=100&sortField=ordinal&sortDir=asc&row=0&searchKeyword=1727483&searchOn=id) focus indicator missing on select language and select format
- [x] 🚨 [1727249](https://axeauditor.dequecloud.com/test-run/b2613a94-fe0b-11ee-96ce-dfcb37e53e3c/issue/6a697dd4-fe27-11ee-bc1f-9f18b5e39269?sortField=severity&sortDir=desc&page=0&issuesCount=&filter%5Bcomponent_number%5D=1&row=0) green check mark validation on form fields lacking alternate text

**Comment by angelinanz** [comment]
This is a duplicate
